packages/cross_file/pubspec.yaml

```diff
@@ -2,18 +2,14 @@ name: cross_file
 description: An abstraction to allow working with files across multiple platforms.
 repository: https://github.com/flutter/packages/tree/main/packages/cross_file
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+cross_file%22
-version: 0.3.2
+version: 0.3.2+1
```

Comment:
I think it's worth making this 0.3.3; it's a more significant change to the package than just a bugfix.